packages/react-scripts/scripts/init.js

```diff
@@ -99,7 +101,24 @@ module.exports = function(
     command = 'npm';
     args = ['install', '--save', verbose && '--verbose'].filter(e => e);
   }
-  args.push('react', 'react-dom');
+  args.push(
+    'redux',
+    'react-redux',
+    'redux-promise',
+    'bootstrap',
+    'jquery',
+    'popper',
+    'prop-types',
+    'enzyme',
+    'enzyme-adapter-react-16',
+    'eslint-config-react-app',
+    'babel-eslint@^7.2.3',
+    'eslint@^4.1.1',
+    'eslint-plugin-flowtype@^2.34.1',
+    'eslint-plugin-import@^2.6.0',
+    'eslint-plugin-jsx-a11y@^5.1.1',
+    'eslint-plugin-react@^7.1.0'
+  );
```

Comment:
Since eslint (and actually enzyme) is a developer dependency rather than a strict dependency, do we want to break this out into two commands?

Reply:
Good idea. I'll make that change